geode-core/src/main/java/org/apache/geode/management/internal/cli/i18n/CliStrings.java

```diff
@@ -861,6 +861,15 @@
       "region-time-to-live-expiration-action";
   public static final String CREATE_REGION__REGIONEXPIRATIONTTLACTION__HELP =
       "Action to be taken on a region that has exceeded the TTL expiration.";
+  public static final String CREATE_REGION__EVICTION_ACTION = "eviction-action";
+  public static final String CREATE_REGION__EVICTION_ACTION__HELP =
+      "The eviction action to apply. Must be either 'destroy' or 'overflow-to-disk'";
+  public static final String CREATE_REGION__EVICTION_MAX_MEMORY = "eviction-max-memory";
+  public static final String CREATE_REGION__EVICTION_MAX_MEMORY__HELP =
+      "Activates LRU eviction based on the region's memory usage specified by this value.";
+  public static final String CREATE_REGION__EVICTION_ENTRY_COUNT = "eviction-entry-count";
+  public static final String CREATE_REGION__EVICTION_ENTRY_COUNT__HELP =
+      "Activates LRU eviction based on the region's entry count specified by this value.";
```

Comment:
When do we stop adding to CliStrings and just define these new strings where they're used in the Command classes? (See the email discussion thread " [Discuss] CliStrings".)